examples/otel-collector-sample-configurations/README.md

```diff
@@ -0,0 +1,25 @@
+# Purpose of this directory
+
+This folder provides sample [custom monitoring configurations](http://cloud/anthos-config-management/docs/how-to/monitor-config-sync-custom)
+for Config Sync. These examples are intended for your convenience. While Config
+Sync strives to keep them updated, always consult the [latest configuration](https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/pkg/metrics/otel.go).
```

Comment:
I think a good way to ensure this stays up to date would be to test it. Can these example templates be exercised/verified in e2e tests?

Reply:
They could, these files serves more as recommendations but it's good to validate them in test. Give me a moment.